readme.md

```diff
@@ -1 +1,86 @@
-Visitor project
+<h1>Visitor project</h1>
+
+The project aims at providing an API for a sample visitor model.
+
+
+<b>Visitor Project Endpoints</b>
+
+<b> GET Request </b>
+**authorisation details to be provided**
+
+url : '/visitor/'
+
+<u> Sample Input </u>
+http -a username:password GET  http://127.0.0.1:8000/visitor/ 
+
+
+<u>Sample Output</u>
+
+HTTP 200 OK
+Allow: GET, POST, HEAD, OPTIONS
+Content-Type: application/json
+Vary: Accept
+
+[
```

Comment:
Do we really need `[` square bracket?? Is the format you have written `JSON` ???